.circleci/config.yml

```diff
@@ -175,7 +186,6 @@ jobs:
 
 workflows:
   workflow_e2e-tester:
-    when: << pipeline.parameters.run_e2e_test >>
```

Comment:
Don't remove! This is needed

Reply:
Ah, my bad. I was trying to run the e2e job locally with circleci CLI, but it was acting up hence removed it. Will add it back!